extras/bmp280/bmp280.h

```diff
@@ -93,7 +94,8 @@ typedef enum {
 typedef struct {
     BMP280_Mode mode;
     BMP280_Filter filter;
-    BMP280_Oversampling oversampling;  // pressure oversampling
+    BMP280_Oversampling oversampling_pressure;
+    BMP280_Oversampling oversampling_temperature;
```

Comment:
Enum `BMP280_Oversampling` was initially create only for pressure oversampling values.  Even though temperature oversampling value is also 3 bits wide the meaning of the enum values are not quite the same. For example from the datasheet temperature oversampling values 2 is used for "Ultra Hi Resolution" pressure measurement. 

I originally didn't provide option for temperature oversampling because the datasheet recommends to select temperature oversampling depending on the desired pressure resolution.